testing/firefox/APKBUILD

```diff
@@ -37,7 +37,7 @@ makedepends="
 	libvorbis-dev
 	libxt-dev
 	libxcomposite-dev
-	llvm5-dev
+	llvm8
```

Comment:
I think we still need -dev for llvm-config

Reply:
Passed without it :)